public/src/Footer/footer.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import './_footer.scss';
+import { AiOutlineFacebook } from 'react-icons/ai';
+import { AiOutlineTwitter } from 'react-icons/ai';
+import { AiOutlineInstagram } from 'react-icons/ai';
+import { AiOutlineYoutube } from 'react-icons/ai';
+
+let footerVars = {
+    nav: [
+        {
+            name: 'Home',
+            link: '#',
+        },
+        {
+            name: 'Store',
+            link: '#',
+        },
+        {
+            name: 'F.A.Q.',
+            link: '#',
+        },
+        {
+            name: 'Our Facilities',
+            link: '#'
+        },
+        {
+            name: 'Contact Us',
+            link: '#',
+        },
+        {
+            name: 'Create Account',
+            link: '#',
+        },
+        {
+            name: 'Login',
+            link: '#',
+        },
+    ],
+    socialMedia: [
+        '#',
+        '#',
+        '#',
+        '#',
+    ],
+};
+
+const Footer = () => {   
+    let styles = {
+        'font-size': 30,
+        'margin-left': 5,
+        'margin-right': 5,
+        'margin-bottom': 5,
+    };
+    return (
+        <div id='footer'>
+            <p id='footer-nav'>
+                <a href={footerVars.nav[0].link}>{footerVars.nav[0].name}</a> • 
```

Comment:
These isn't a stopper but I'd challenge you to figure out a way to not copy past this code.. There should be a way you can loop over this